x/token/keeper/msg_server_test.go

```diff
@@ -662,200 +664,517 @@ func (s *KeeperTestSuite) TestMsgRevokePermission() {
 	}
 }
 
-func (s *KeeperTestSuite) TestMsgMint() {
+func (s *KeeperTestSuite) TestMsgMintVerifyEvent() {
```

Comment:
Why don't you integrate all MsgMint-related tests in one test function instead of Instead of dividing it into `VerifyEvent`, `PositiveCase`, and `NegativeCase`.

Reply:
I divided them because I wanted to have focused verification logic.
By doing so, it gets easier to understand what to verify for happy path and negative path.
I divide the event test with TestXXVerifyEvent because I have tests that only focus on the event itself. And I want removes listing all the events which takes lots of code spaces for every positive cases as well.